static/app/components/contextPickerModal.tsx

```diff
@@ -62,6 +63,7 @@ type Props = ModalRenderProps & {
    * on which the modal was opened
    */
   comingFromProjectId?: string;
+  configs: Integration[];
```

Comment:
@NisanthanNanthakumar I think `integrationConfigs` is a better and more descriptive name for the field